static-crawling/static-crawling_assignment.py

```
@@ -0,0 +1,176 @@
+
+# ### 1. 필요한 라이브러리 설치 및 임포트
+# 먼저, 웹 크롤링을 위해 필요한 라이브러리들을 설치하고 임포트합니다.
+# 
+# - bs4: BeautifulSoup 라이브러리는 HTML/XML 페이지를 파싱하여 데이터를 쉽게 추출할 수 있게 도와줍니다.
+# - requests: HTTP 요청을 보내 웹 페이지의 HTML을 받아오는 라이브러리입니다.
+# - pandas: 데이터를 표 형태로 처리하고, csv 파일로 저장하는 데 사용됩니다.
+
+# In[10]:
+
+
+get_ipython().run_line_magic('pip', 'install bs4')
```

Ruff (0.13.1)

Undefined name `get_ipython`

(F821)